### PR TITLE
[OSDOCS-7449]: Clarify 'pausing reconciliation' docs for hosted control planes

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/OSDOCS-7449

Link to docs preview: https://63491--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-managing#hosted-control-planes-pause-reconciliation_hcp-managing

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by jparrill — COMMENTED** [review]
/lgtm

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Comment by GroceryBoyJr** [comment]
@lahinson I wish there were an easy way to include your added sentence as a NOTE, but I'm not sure the detail is significant enough to need a note? (and I'm not sure theres any customer case reduction value in such a change?)

**Comment by lahinson** [comment]
@GroceryBoyJr Thanks for the review! As I understand it, the sentence isn't significant enough to be highlighted as a note.

File: modules/hosted-control-planes-pause-reconciliation.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ If you are a cluster instance administrator, you can pause the reconciliation of
 
 .Procedure
 
-. To pause reconciliation for a hosted cluster and hosted control plane, populate the `pausedUntil` field of the `HostedCluster` resource, as shown in the following examples:
+. To pause reconciliation for a hosted cluster and hosted control plane, populate the `pausedUntil` field of the `HostedCluster` resource, as shown in the following examples. In the examples, the value that is specified for the `pausedUntil` field is wrapped into the command that follows it.
```
Comment (bergerhoffer) [inline modules/hosted-control-planes-pause-reconciliation.adoc:13]: Until I looked at the code block to see what was going on, I didn't know what "wrapped into the command" meant.

I'd recommend rewriting to be clearer what is meant. Maybe something like:

```suggestion
. To pause reconciliation for a hosted cluster and hosted control plane, populate the `pausedUntil` field of the `HostedCluster` resource, as shown in the following examples. In the examples, the value for `pausedUntil` is defined in an environment variable prior to the command.
```

Also side note for another time, the below commands use "kubectl", but commands in our docs should use "oc".

File: modules/hosted-control-planes-pause-reconciliation.adoc [hunk 2]
```diff
@@ -22,7 +22,7 @@ kubectl patch -n <hosted-cluster-namespace> hostedclusters/<hosted-cluster-name>
 +
 The reconciliation is paused until the specified time is passed.
 +
-** To pause the reconciliation indefinitely, pass a Boolean value of `true`:
+** To pause the reconciliation indefinitely, pass a Boolean value of `true`.
```
Comment (bergerhoffer) [inline modules/hosted-control-planes-pause-reconciliation.adoc:25]: I think this should stay a colon

**Review by bergerhoffer — COMMENTED** [review]
A few suggestions prior to merging!

File: modules/hosted-control-planes-pause-reconciliation.adoc (hunk 1 above)
Comment (lahinson) [reply modules/hosted-control-planes-pause-reconciliation.adoc:13]: @jparrill Would it affect the accuracy of the commands here if I changed `kubectl` to `oc`?

**Pull request merged**